stob/ad.c

```diff
@@ -1093,7 +1092,7 @@ static uint32_t stob_ad_write_map_count(struct m0_stob_ad_domain *adom,
 		frag_size = min_check(m0_ivec_cursor_step(&it), grp_size);
 		M0_ASSERT(frag_size > 0);
 		M0_ASSERT(frag_size <= (size_t)~0ULL);
-		M0_LOG(M0_DEBUG, "frag_size=0x%lx", frag_size);
+		M0_LOG(M0_DEBUG, "frag_size=0x%"PRId64, frag_size);
```

Comment:
PRIx64 is needed, or,
remove the "0x" in the fmt string.

Reply:
Fixed.